ion/src/main/java/com/fasterxml/jackson/dataformat/ion/EnumAsIonSymbolSerializer.java

```diff
@@ -49,7 +50,11 @@ public EnumAsIonSymbolSerializer() {
     @Override
     public void serialize(Enum<?> value, JsonGenerator g, SerializerProvider provider) throws IOException {
         if (IonGenerator.class.isAssignableFrom(g.getClass())) {
-            ((IonGenerator) g).writeSymbol(value.name());
+             String valueString = provider.isEnabled(SerializationFeature.WRITE_ENUMS_USING_TO_STRING)
```

Comment:
nit: looks like there is an extra space on this line

Reply:
Will fix